spread.yaml

```diff
@@ -962,27 +942,16 @@ suites:
             # Install the snapd built
             dpkg -i "$SPREAD_PATH"/../snapd_*.deb
 
-            #shellcheck source=tests/lib/nested.sh
-            . "$TESTSLIB/nested.sh"
-            nested_prepare_env
-            nested_create_core_vm
+            "$TESTSTOOLS"/nested-state prepare
+            "$TESTSTOOLS"/nested-state build-image core
         prepare-each: |
             tests.backup prepare
-
-            #shellcheck source=tests/lib/nested.sh
-            . "$TESTSLIB/nested.sh"
-            nested_start_core_vm
-            nested_wait_for_snap_command
```

Comment:
Ok, this is fine since we now wait inside nested_start_core_vm (nested_start_core_vm_unit).

Reply:
right